### ISSUE TITLE
Block, Block Section - Ensure images provide informative alternative text - (2036569802)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5566]
### Module: 
 08 Block &amp; Block Section 

### Violation: 
 Ensure images provide informative alternative text 

![image](https://user-images.githubusercontent.com/5023024/202800510-16827aa7-e6a2-4a7c-9ccd-fe5efa578113.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036569802 

### Severity: 
 10 

### Description: 
 [Issue]
There are meaningful svg images without textual equivalents. Examples include:

- Success icon
- Invalid icon
- Loading

[User Impact]
Screen reader users will not receive an accurate impression of the contents of these images.

[Code Reference] 
 ```
<calcite-icon class="status-icon valid" aria-hidden="true" icon="check-circle" scale="m" calcite-hydrated="">
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 24 24" width="100%" xmlns="http://www.w3.org/2000/svg">
.....
</svg>
</calcite-icon>
 ``` 

### Note: 
 [Suggestion]
Ensure images provide informative alternative text. Textual equivalents must be both concise and descriptive. In this case remove aria-hidden="true" from the ``` <Calcite> ``` element. In the case of svg elements, it is recommended to add role="img", ``` <title> ``` with the text equivalent and aria-labelledby set to the id of the title element

[Compliant Code Example] 
 ```
<calcite-icon class="status-icon valid" aria-hidden="true" icon="check-circle" scale="m" calcite-hydrated="">
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 24 24" width="100%" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="title">
<title id="title">Success</title>
.....
</svg>
</calcite-icon>
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Similar to the need for #5598
